### PR TITLE
drop serving v1beta1 and v1alpha1 APIs

### PR DESCRIPTION
**Pull request by dprotaso**
Fixes https://github.com/knative/serving/issues/8638

## Proposed Changes 

- Drops v1alpha1 & v1beta1

### CODE REVIEW

File: docs/reference/api/serving.md [hunk 1]
```diff
@@ -6,9 +6,6 @@
 <li>
 <a href="#serving.knative.dev%2fv1alpha1">serving.knative.dev/v1alpha1</a>
 </li>
```
Comment (mattmoor) [inline docs/reference/api/serving.md:8]: Hmm, is this because I left the empty thing?

Comment (dprotaso) [reply docs/reference/api/serving.md:8]: I'm guessing so I ran the tool

Comment (dprotaso) [reply docs/reference/api/serving.md:8]: I can manually drop this

**Comment by mpetason** [comment]
I'm not sure why it is failing the trailing whitespace test. I didn't see extra space where the updates were made.

**Comment by dprotaso** [comment]
@mpetason the docs are autogenerated so it's in the tooling.

the whitespace action isn't blocking so once this gets a `/lgtm` it'll merge

**Pull request merged**